core/src/main/java/tech/tablesaw/joining/DataFrameJoiner.java

```diff
@@ -355,10 +385,21 @@ private Selection selectionForColumn(Column<?> valueColumn, int rowIndex, Index
               + valueColumn.type());
     }
   }
-
-  /** Create a big multicolumn selection for all join columns in the given table. */
+  /**
+   * Create a big multicolumn selection for all join columns in the given table. Joins two tables.
+   *
+   * @param table1 the table which used to generate Selection.
+   * @param ri row number of row in table 1.
+   * @param selectionSize max size in table 1.
```

Comment:
It seems to be missing the description of argument 
   
    List<Index> indexes

I don't understand what the description of joinColumnIndexes is telling me. It appears that it should read
   the column indexes (plural) of table1
but what column indexes, and what are they used for?

Reply:
Thanks for your review:) 

> but what column indexes, and what are they used for?


It is the index in tables' column list, representing "join key" for joining. and will be used for cross join and catcat table.

> I don't understand what the description of joinColumnIndexes is telling me. It appears that it should read
the column indexes (plural) of table1


Before I change, it would read the column index of table1; 
After I change, It can be the column index of table2 for better performance. **I will change the variable name and JAVA doc tomorrow for more clear understanding.**